### DISCUSSION TITLE
Update in Umbrel OS

### DISCUSSION THREAD

**Discussion by reddai75** [ghostfolio/ghostfolio#4039] [Q&A]
the last version of ghostfolio available on the umbrel app store is 2.105, is there any chance to have the latest version there ? That version has the bug of the historycal data ...

**Comment by dtslvr** [comment]
Hi @reddai75

Please direct your request to the [Umbrel community](https://community.umbrel.com) or consider creating a PR based on [this example](https://github.com/getumbrel/umbrel-apps/pull/1115).
